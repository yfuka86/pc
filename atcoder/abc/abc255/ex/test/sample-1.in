5 3
2 2 3
3 3 4
5 1 5
