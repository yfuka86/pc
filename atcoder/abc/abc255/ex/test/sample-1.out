10
15
50
